cmd/codegen/generator/go/templates/module_objects.go

```diff
@@ -419,6 +419,14 @@ func (spec *parsedObjectType) concreteFieldTypeCode(typeSpec ParsedType) (*State
 		if typeSpec.alias != "" {
 			s.Id(typeSpec.alias)
 		} else {
+			tp := typeSpec.GoType()
+			if basic, ok := tp.(*types.Basic); ok {
+				if basic.Kind() == types.Invalid {
+					s.Id("any")
+					break
+				}
```

Comment:
This bit seems interesting - is it to prevent codegen from blowing up on an unknown type? Maybe because it gets figured out on another pass?

Reply:
Yup, that's the behavior in [`c50cb10` (#7831)](https://github.com/dagger/dagger/pull/7831/commits/c50cb109a0a71c8d92d3e97da6c2e340c2be16ce). It produces invalid code, but the code wouldn't have compiled in the first place - this means `dagger develop` can at least succeed, so you can actually fix the issues, instead of having to just "guess".